network/firewall.py

```diff
@@ -0,0 +1,396 @@
+import logging
+import json
+from typing import List, Dict
+
+import network.network as nw
+from nftables import Nftables
+
+logger = logging.getLogger(__name__)
+
+nft = Nftables()
```

Comment:
Can we avoid this global variable ?

Reply:
I'm assuming you're talking about the nft one, not the logger. Yeah that one should probably just move into the execute_json_nft_commands(). Before I had a single function to take care of executing the commands, having the configuration of on top made more sense.

If you're talking about the logger, it's because it's done like that in all files.